infrastructure/modules/persistent-volumes/efs/variables.tf

```diff
@@ -43,7 +43,11 @@ variable "csi_driver" {
 # VPC infos
 variable "vpc" {
   description = "AWS VPC info"
-  type        = any
+  type = object({
+    id          = string
+    cidr_blocks = list(string)
```

Comment:
I think it could be better to have `set` instead of `list`

Reply:
done